ilc/server/registry/Registry.js

```diff
@@ -93,4 +98,23 @@ module.exports = class Registry {
 
         return res.data;
     };
+
+    /**
+     * Returns new object with routes list for current domain
+     * and without "supportedDomains" data and property "domainId" inside every "route"
+     */
+    #filterConfigByDomain = (config, domain) => {
```

Comment:
Can we avoid fn params modification & rather create new object?

Reply:
I decided to use `deepmerge` as a clone-tool. What do you think about it?
https://github.com/namecheap/ilc/pull/278/files#diff-fe500a7438ccd9d0cbb535ee88ca3a0ba5f994e030980275f9821b4e96d0312dR25